Prac 5/colour_names.py

```diff
@@ -0,0 +1,16 @@
+colours = {'rebeccapurple': '#663399', 'darkorange': '#ff8c00', 'cyan2': '#00eeee',
+          'cornflowerblue': '#6495ed', 'darkseagreen': '#8fbc8f', 'deeppink1': '#ff1493',
+          'darkslateblue': '#483d8b', 'dodgerblue1': '#1e90ff', 'goldenrod': '#daa520',
+          'lightslateblue': '#8470ff'}
+
+colour_name = input('Please enter the name of a colour: ')
```

Comment:
Could use .lower() here to ensure the user finds thier colour even if they use uppercases